### PR TITLE
Change old-style class to new-style & remove unused function

### PR DESCRIPTION
**Pull request by vmuriart**
Change old-style class to new-style

py3 interpreted old code as new-style classes. py2 should match its behavior.
Remove unused hash function.
Update config for travis/tox

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,23 +1,31 @@
...
+    - pip install tox coveralls
 
-script: py.test --cov=sqlparse
+script:
+    - tox
 
 after_success:
-  coveralls
+    - coveralls
+    - bash <(curl -s https://codecov.io/bash)
```
Comment (andialbrecht) [inline .travis.yml]: What does this bash line do? If it's not important in some way I'd prefer to leave this out. I really don't like to run any bash scripts fetched somewhere from the internet.

**Comment by andialbrecht** [comment]
Thanks!

File: .travis.yml (hunk 1 above)
Comment (vmuriart) [inline .travis.yml]: It uploads the coverage report to `codedov` which has a friendlier UI to review coverage. 
I just realized I got their **old** upload script. This was their default setup script to add to `travis-ci` when you first open the account. But on [their documentation](https://github.com/codecov/example-python#-travis-ci) they have a much better way to do it.

I'll attach the change to the next `pr`

Comment (andialbrecht) [inline .travis.yml]: Cool, thanks! (However, iat the end of the day there's little difference between "curl"ing a bash script or "pip installing" a Python module, but it _feels_ better :smile:)

**Pull request merged**